master/buildbot/data/connector.py

```diff
@@ -148,3 +152,56 @@ def allEndpoints(self):
                               type=str(v.rtype.entityType.name),
                               type_spec=v.rtype.entityType.getSpec()))
         return paths
+
+    @functools.lru_cache(1)
+    def genGraphQLSchema(self):
```

Comment:
I think we should use `snake_case` for new private APIs. This will introduce some inconsistency short term, but long term we would be moving to the same naming as the rest of the python ecosystem.